platform-sdk/swirlds-merkledb/src/main/java/com/swirlds/merkledb/collections/AbstractLongList.java

```diff
@@ -126,15 +127,23 @@ public abstract class AbstractLongList<C> implements LongList {
      */
     protected final long reservedBufferLength;
 
+    protected final FileSystemManager fileSystemManager;
```

Comment:
Would it make more sense to have this value as part of `LongListDisk` instead of here?
It seems like this is only needed in that particular implementation.

Reply:
@MiroslavGatsanoga Joseph is right. `LongListDisk` is the only implementation of `LongList` that works with disk. Please, move `FileSystemManager` there.

Reply:
Moved `FileSystemManager` to `LongListDisk`